src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1488,5 +1488,9 @@ internal enum ErrorCode
         ERR_PatternWrongGenericTypeInVersion = 8314,
 
         #endregion diagnostics introduced for C# 7.1
+
+        #region diagnostics introduced for C# 7.2
+        ERR_FeatureNotAvailableInVersion7_2 = 8320,
```

Comment:
This is going into 15.6 branch. 7.1 is not closed yet, so I left some buffer. (Just today I had to use a new error code for a 7.1 fix).